infra/cifuzz/fuzz_target.py

```diff
@@ -121,6 +124,32 @@ def is_reproducible(self, test_case):
         return True
     return False
 
+  def is_crash_valid(self, test_case):
+    """Checks if a crash was introduced by the pull request.
```

Comment:
change the comment to say something like "Checks if a crash is reproducible, and if it is, whether it's a new regression that cannot be reproduced with the latest OSS-Fuzz build."

Reply:
Added.